mods/persistence/modules/mob/living/carbon/human/logout.dm

```diff
@@ -0,0 +1,8 @@
+/mob/living/carbon/human/Logout()
+	. = ..()
+	for(var/obj/content in loc.contents)
+		if(istype(content, /obj/structure/bed))
+			SetStasis(20, STASIS_SLEEP) // beds are better.
+			return
+	//Apply sleeping stasis.
+	SetStasis(10, STASIS_SLEEP)
```

Comment:
Can definitely see players logging out to stasis while their ally drags them out of combat. Or medical shouting for any players who need treatment to log out to make triage easier. Maybe put a timer on it?

Reply:
Maybe make interacts like that bring you out of stasis? That way medical doesn't find someone SSD and gets confused why medicine isn't working (because it's much slower)

Reply:
I added a timer, doing medical on a sleeping patient (SSD) is pretty edge case. Future PR for polish.